api/swagger/src/definitions/definitions.yaml

```diff
@@ -27,23 +27,19 @@ UpdateEntity:
 Artist:
   properties:
     id:
-      type: integer
-      format: int32
+      type: string
```

Comment:
@kordianbruck  : yes in Sequelize Primary Key is defined as UUID .
Eg:[{"name":"Frank Zappa","id":"b9dcc333-4f23-44c1-a521-18a3fc1270b8","artist_type":"musician"......
Hence, the data type of id in Swagger Definition has to be string .